README.md

```diff
@@ -0,0 +1,37 @@
+<!---
+   Copyright 2021 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+<img src="./images/logo.png" alt="Eiffel Community Repository Template" width="350"/>
```

Comment:
We do not have this resource yet it seems.

Reply:
I'll amend and add that logo as well.

Reply:
Thought about this again and perhaps I shouldn't include the logo since the logo in the template repository says repo template.

The maintainers should fix a logo for the repo.